test/performance/odp_l2fwd.c

```diff
@@ -1243,10 +1494,17 @@ static void usage(char *progname)
 	       "  -p, --packet_copy       0: Don't copy packet (default)\n"
 	       "                          1: Create and send copy of the received packet.\n"
 	       "                             Free the original packet.\n"
-	       "  -y, --pool_per_if       0: Share a single pool between all interfaces (default)\n"
+	       "  -y, --pool_per_if       Number of packet or packet vector pool per interface.\n"
+	       "                          0: Share a single pool between all interfaces (default)\n"
 	       "                          1: Create a pool per interface\n"
 	       "  -n, --num_pkt <num>     Number of packets per pool. Default is 16k or\n"
 	       "                          the maximum capability. Use 0 for the default.\n"
+	       "  -u, --vector_mode       Enable vectors mode.\n"
+	       "                          Supported only with scheduler packet input modes (1-3).\n"
```

Comment:
It would be good to check this also in code. Now the result will be a random crash if unsupported mode is selected.

Reply:
Will add a check.